### ISSUE TITLE
Can I use logical or in query?

### ISSUE THREAD

**Issue by zyzsixsixsix** [pcarbonn/pyDatalog#32]
I only found &，but I want to use logical or to get two conds union.How can achieve it?

**Comment by mkuijn-cb** [comment]
https://stackoverflow.com/questions/56623804/disjunctions-in-datalog

**Comment by pcarbonn** [comment]
No, you can't.  This is not possible in standard datalog nor prolog.
